ote/src/cljs/ote/app/controller/transport_service.cljs

```diff
@@ -364,7 +368,7 @@
                                          (filter #(= (::t-operator/id %)
                                                      (::t-service/transport-operator-id response)))
                                          first))
-          app (country-code->country app (:transport-service app))]
+          app (str-cc->keyword-cc app (:transport-service app))]
```

Comment:
`app` object is refined and enriched step by step ==> looks like a good place for a threading macro, but this is ok of course as well.